frontend/src/component/feature/FeatureForm/FeatureForm.tsx

```diff
@@ -197,6 +197,7 @@ const FeatureForm: React.FC<IFeatureToggleForm> = ({
                     </StyledInputDescription>
                 }
             />
+            // TODO: this can be removed after new create flag flow goes live
             <FeatureProjectSelect
                 value={project}
                 onChange={(projectId) => {
```

Comment:
This is using create-toggle, but this is the old creation component that will be removed.